.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -1,4 +1,8 @@
+---
 contact_links:
-- name: Get help on Discord
-  url: https://lmms.io/chat/
-  about: Need help? Have a question? Reach out to other LMMS users on our Discord server!
+  - name: Get help on Discord
+    url: https://lmms.io/chat/
+    about: >
+      Need help? Have a question?
+      Reach out to other LMMS users
+      on our Discord server!
```

Comment:
```suggestion
    about: Need help? Have a question? Reach out to other LMMS users on our Discord server!
```

Reply:
Thanks, though this one here will cause yamllint to fail because of more than 80 characters per line (I don't know how to increase that number).